cfme/tests/services/test_myservice.py

```diff
@@ -147,8 +147,13 @@ def test_service_link(appliance, context, myservice):
     """
     service_name, vm_name = myservice
     with appliance.context.use(context):
+        # TODO: Update to nav to MyService first to click entity link when widget exists
         myservice = MyService(appliance, name=service_name, vm_name=vm_name)
-        view = navigate_to(myservice, 'VMDetails')
-        view.summary('Relationships').click_at('Service')
+        vm = appliance.provider_based_collection(coll_type='vms', provider=provider).instantiate(
+            name=myservice.vm_name,
+            provider=provider
+        )
+        view = navigate_to(vm, 'Details')
+        view.entities.summary('Relationships').click_at('Service')
         new_view = myservice.create_view(MyServiceDetailView)
```

Comment:
I would use `wait` arg.

Reply:
Again, I'm explicitly asserting against `view.wait_displayed()`, so I don't care to have create_view wait for me.